src/qservkafka/handlers/kafka.py

```diff
@@ -16,7 +16,9 @@
 
 
 @kafka_router.subscriber(
-    config.job_run_topic, group_id=config.consumer_group_id
+    config.job_run_topic,
+    auto_offset_reset="earliest",
```

Comment:
Can this go in the [router config](https://github.com/lsst-sqre/qserv-kafka/pull/25/files#diff-1da46c4b8f3f0d8a78d5a95dcc1f483d0a6206304b06a7054304b7088f2e19f8R19), and would that make apply it to all subscribers (even though there's only one now)?

Reply:
Looks like it can't, nevermind!